Libplanet/Net/Swarm.cs

```diff
@@ -1578,9 +1579,15 @@ CancellationToken cancellationToken
                         }
                         else
                         {
+                            FillBlocksAsyncFailed.Set();
```

Comment:
I guess this statement isn't necessary because we'll set this event [here](https://github.com/planetarium/libplanet/pull/686/files#diff-9501da6c2e844a81d5e3ad3d3d5456ccR1588).

Reply:
I thought so, but the [catch block](https://github.com/planetarium/libplanet/pull/686/files#diff-9501da6c2e844a81d5e3ad3d3d5456ccR1586) doesn't seem to catch the re-thrown `TimeoutException`. 😢

Reply:
👌